libflux/flux-core/src/formatter/tests.rs

```diff
@@ -7,11 +7,7 @@ use pretty_assertions::assert_eq;
 
 fn assert_unchanged(script: &str) {
     let output = format(script).unwrap();
-    assert_eq!(
-        script, output,
-        "\n EXPECTED: \n {} \n OUTPUT: \n {} \n",
-        script, output
-    );
+    assert_eq!(script, output);
```

Comment:
@scbrickley I actually think this case is different. I think we should leave these changes in place. The normal diff output of the Rust assert_eq macro flattens the strings onto a single line using escaped newlines and tabs instead of literals. I.e. it looks like this `a = b\nc = d` which can be hard to read, specially when checking for indentation is correct across multiple lines. Since this test case is about testing the formatting I think it makes sense to print the expected and output as string literals without any escaping like this:

```
a = b
c = d
```

Both outputs are valuable since the formatted output makes it easy to visualize that your change worked or didn't. The escaped output makes it easy to find off-by-one errors with whitespace that otherwise are hard to spot. 

Thoughts?

Reply:
Good points. I guess I hadn't considered that formatting presented special concerns that we don't have in other areas of `libflux`. I'm for adding it back in if you think that's best.